File: documentation-examples-kotlin/src/kotlin/dev/typr/foundations/docs/postgresql/BinaryTypes.kt
package dev.typr.foundations.docs.postgresql

import dev.typr.kotlinfoundations.*
import dev.typr.kotlinfoundations.data.*

@Suppress("unused")
class BinaryTypes {
    //start
    val bytesType: PgType<ByteArray> = PgTypes.bytea
    //stop
}

File: documentation-examples-scala/src/scala/dev/typr/foundations/docs/postgresql/BinaryTypes.scala
package dev.typr.foundations.docs.postgresql
import dev.typr.scalafoundations.*
import dev.typr.scalafoundations.data.*



@SuppressWarnings(Array("unused"))
object BinaryTypes:
  //start
  val bytesType: PgType[Array[Byte]] = PgTypes.bytea
  //stop
